app/code/Codifi/CustomerRequest/Model/Config/Backend/Archive.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+declare(strict_types=1);
+
+namespace Codifi\CustomerRequest\Model\Config\Backend;
+
+use Magento\Framework\App\Config\Value;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Registry;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\Cache\TypeListInterface;
+use Magento\Framework\App\Config\ValueFactory;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Cron\Model\Config\Source\Frequency;
+use \Exception;
+
+/**
+ * Class Archive
+ * @package Codifi\CustomerRequest\Model\Config\Backend
+ */
+class Archive extends Value
+{
+    /**
+     * Cron string path
+     */
+    const CRON_CODIFI_STRING_PATH = 'crontab/archive/jobs/codifi_customerrequest_cron_archivecronmodel/schedule/cron_expr';
+
+    /**
+     * Value factory
+     *
+     * @var ValueFactory
+     */
+    protected $_configValueFactory;
```

Comment:
1. Must be private;
2. Property names MUST NOT be prefixed with a single underscore;
3. You must use ResourceModel or Repository to save model;